README.md

```diff
@@ -69,7 +69,7 @@ export class JSDOM {
     return JSDOM.#finalizeFactoryCreated(new JSDOM(body, options), "fromURL");
   }
   
-  static fromFile(filename, options = {}) {
+  async static fromFile(filename, options = {}) {
```

Comment:
I’m pretty sure it is actually `static async`; guess we’ll need another PR to correct it.

Reply:
I'm happy if that's the case, but I just copied from the example above.